proposals/integration-test-cases.md

```diff
@@ -0,0 +1,103 @@
+# Integration test case suggestions
+This document gives an overview of possible integration test cases, provides some useful links and a specification how to write the docstring for a test case.
```

Comment:
I'd make it more obvious that docstring is a word in the Python realm. A casual reader might not get it on a first look.